pkg/types/openstack/platform.go

```diff
@@ -28,6 +28,14 @@ type Platform struct {
 	// Existing Floating IP to associate with the OpenStack load balancer.
 	LbFloatingIP string `json:"lbFloatingIP"`
 
+	// APIIntIP
+	// IP in the machineCIDR to use for api-int.
+	APIIntIP string `json:"apiIntIP"`
```

Comment:
FYI the values that the (merged) Baremetal `PlatfomStatus` support uses `APIServerInternalIP` and `NodeDNSIP` names:

https://github.com/openshift/api/pull/348/files

And so does the current (unmerged) OpenStack equivalent: https://github.com/openshift/api/pull/374

(not sure how much sense does it make for these to align)

Reply:
Since I needed to rebase this on what the baremetal platform landed I named the vars similar to theirs.